### PR TITLE
Enabling case worker sorting

### PR DESCRIPTION
**Pull request by kth13**
## What does this pull request do?

Enables case worker sorting in both probation practitioner and service provider dashboards. Currently its been disabled for all the users. The sorting happens based on the caseworker name.

## What is the intent behind these changes?

The dashboard for both probation practitioner and service provider has got a caseworker column. As the number of referrals growing , there is a need to enable sorting so that users can sort and traverse to the intended records sooner.

### CODE REVIEW

File: server/config.ts [hunk 1]
```diff
@@ -134,6 +134,7 @@ export default {
   },
   dashboards: {
     probationPractitioner: {
+      percentageOfPaginationUsers: Number(get('SP_PERCENTAGE_OF_DASHBOARD_PAGINATION_USERS', '0')),
```
Comment (sldblog) [inline server/config.ts:137]: _nit_ It doesn't look like `dashboards.probationPractitioner.percentageOfPaginationUsers` is used anywhere (PP dashboards are always on) but it's not a blocker for this change

File: helm_deploy/values-preprod.yaml [hunk 2]
```diff
@@ -14,5 +14,9 @@ generic-service:
     GA_ID: G-08D8T0RZZR
     FEATURE_SP_REPORTING_ENABLED: true
     SP_PERCENTAGE_OF_DASHBOARD_PAGINATION_USERS: 100
+    SP_MY_CASES_PAGE_SIZE: 10
+    SP_OPEN_CASES_PAGE_SIZE: 10
+    SP_UNASSIGNED_CASES_PAGE_SIZE: 10
+    SP_COMPLETED_PAGE_SIZE: 10
```
Comment (sldblog) [inline helm_deploy/values-preprod.yaml:20]: _nit_ Did you mean to put this in dev? In preprod, we usually want to replicate the traffic/volumes/load as on production

**Review by sldblog — APPROVED** [review]
LGTM 👍 No blockers, couple of questions

File: server/config.ts (hunk 1 above)
Comment (kth13) [reply server/config.ts:137]: Ok sure David, I will remove it.

File: helm_deploy/values-preprod.yaml (hunk 2 above)
Comment (kth13) [reply helm_deploy/values-preprod.yaml:20]: Dev already the same config and I thought keeping the same thing in pre-prod. In prod, the config is

**Pull request merged**